packages/lisk-p2p/test/unit/peer_selector.ts

```diff
@@ -52,54 +52,60 @@ describe('peer selector', () => {
 				peerList = initializePeerList();
 			});
 
+			it('should return an array without optional arguments', () => {
+				return expect(selectPeers(peerList)).to.be.an('array');
+			});
+
 			it('should return an array', () => {
-				return expect(selectPeers(peerList, option)).to.be.an('array');
+				return expect(selectPeers(peerList, selectionParams)).to.be.an('array');
 			});
 
 			it('returned array should contain good peers according to algorithm', () => {
-				return expect(selectPeers(peerList, option))
+				return expect(selectPeers(peerList, selectionParams))
 					.and.be.an('array')
 					.and.to.be.eql(goodPeers);
 			});
 
 			it('return empty peer list for no peers as an argument', () => {
-				return expect(selectPeers([], option))
+				return expect(selectPeers([], selectionParams))
 					.and.be.an('array')
 					.and.to.be.eql([]);
 			});
 
 			it('should return an array having one good peer', () => {
-				return expect(selectPeers(peerList, option, 1))
+				return expect(selectPeers(peerList, selectionParams, 1))
 					.and.be.an('array')
 					.and.of.length(1);
 			});
 
 			it('should return an array having 2 good peers', () => {
-				return expect(selectPeers(peerList, option, 2))
+				return expect(selectPeers(peerList, selectionParams, 2))
 					.and.be.an('array')
 					.and.of.length(2);
 			});
 
 			it('should return an array having all good peers', () => {
-				return expect(selectPeers(peerList, option, 0))
+				return expect(selectPeers(peerList, selectionParams, 0))
 					.and.be.an('array')
 					.and.of.length(3);
 			});
 
 			it('should return an array having all good peers ignoring requested negative number of peers', () => {
-				return expect(selectPeers(peerList, option, -1))
+				return expect(selectPeers(peerList, selectionParams, -1))
```

Comment:
This test case might be overkill. The caller should never pass a negative number; if they do, then we have a much bigger problem in another part of the code. We can never test every input so we should focus on inputs that are more relevant to program behaviour.